lib/eblox/data/post.ex

```diff
@@ -29,12 +63,15 @@ defmodule Eblox.Data.PostFSM do
   end
 
   def on_transition(:read, :parse, nil, %{content: content} = payload) do
-    case Md.parse(content) do
-      %Md.Parser.State{} = parsed ->
+    case EbloxParser.parse(content) do
+      {_, %Md.Parser.State{} = parsed} ->
+        {html, properties} = Md.generate(parsed, walker: &Walker.prewalk/2, format: :none)
```

Comment:
I love the simplicity of this solution.

What is the `format` option? We may require formatting for text in `<code>` and `<pre>` tags.

Reply:
> I love the simplicity of this solution.

Well, it took me several updates to both `XmlBuilder` and `Md` libs :)

> What is the `format` option? We may require formatting for text in `<code>` and `\<pre\>` tags.

It defines what the generated XML looks like. https://hexdocs.pm/xml_builder_ex/XmlBuilder.html#generate/2

`:none` means no indentation, no whitespace; it does not touch the inner text of tags, e. g. the content of tags.